# Information about a local topic
# and the remotes that are subscribed to it.
string               topic
string               datatype
TopicRemoteDetails[] remotes